### DISCUSSION TITLE
Can't start Zigbee2MQTT container

### DISCUSSION THREAD

**Discussion by mwthrane** [Koenkk/zigbee2mqtt#12514] [General]
Hello 

I've been running Z2M for a few months and now i just wanted to update it with:
`sudo docker pull koenkk/zigbee2mqtt:latest`
`sudo docker rm -f Zigbee2MQTT`

After the update i wanted to start the container again, but i'm met with this:

```
sudo docker logs Zigbee2MQTT
Creating configuration file...
cp: can't create '/app/data/configuration.yaml': Permission denied
```

The container keeps restarting. 

Im starting the container using
```
sudo docker run -d \
   --name=Zigbee2MQTT \
   --restart=unless-stopped \
   -p 8080:8080 \
   -v $(pwd)/data:/app/data \
   -v /run/udev:/run/udev:ro \
   --device=/dev/ttyACM0 \
   --user 1001:1001 \
   --group-add dialout \
   -e TZ=Europe/Amsterdam \
   koenkk/zigbee2mqtt
```

The configuration dir looks like this:

```
thrane@docker01:~/Z2M/app/data$ pwd
/home/thrane/Z2M/app/data
thrane@docker01:~/Z2M/app/data$ la -la
total 112
drwxrwxr-x  4 thrane thrane  4096 May 17 13:29 .
drwxrwxr-x  3 thrane thrane  4096 May 17 13:52 ..
-rw-rw-r--  1 thrane thrane  7548 May 17 13:03 configuration.yaml
-rw-rw-r--  1 thrane thrane  3269 Feb  5 14:55 configuration.yaml.bak
drwxr-xr-x  2 root   root    4096 May 17 13:24 data
-rw-r--r--  1 root   root   70977 May 17 13:10 database.db
drwxr-xr-x 12 root   root    4096 May 17 13:10 log
-rw-r--r--  1 root   root   10838 May 17 13:09 state.json
thrane@docker01:~/Z2M/app/data$
```

Can anyone help?
Thanks.

**Comment by alanmgomes** [comment]
> The configuration dir looks like this:
> 
> ```
> thrane@docker01:~/Z2M/app/data$ pwd
> /home/thrane/Z2M/app/data
> thrane@docker01:~/Z2M/app/data$ la -la
> total 112
> drwxrwxr-x  4 thrane thrane  4096 May 17 13:29 .
> drwxrwxr-x  3 thrane thrane  4096 May 17 13:52 ..
> -rw-rw-r--  1 thrane thrane  7548 May 17 13:03 configuration.yaml
> -rw-rw-r--  1 thrane thrane  3269 Feb  5 14:55 configuration.yaml.bak
> drwxr-xr-x  2 root   root    4096 May 17 13:24 data
> -rw-r--r--  1 root   root   70977 May 17 13:10 database.db
> drwxr-xr-x 12 root   root    4096 May 17 13:10 log
> -rw-r--r--  1 root   root   10838 May 17 13:09 state.json
> thrane@docker01:~/Z2M/app/data$
> ```
> 
> Can anyone help? Thanks.

Looks like a permission problem. U are using a root user to start the container, make sure the user root is on this group 'thrane'.

> -rw-rw-r--  1 thrane thrane  7548 May 17 13:03 configuration.yaml

U can check this by typing this comand:

`cat /etc/group | grep thrane`

**Comment by ankar** [comment]
The problem could be due to the SELinux permissions.
A quick way to check is to reduce temporary the mode to "permissive":
```
$ sestatus
$ sudo setenforce Permissive
```
